lib/core/src/lib/snackbar-content/snackbar-content.component.html

```diff
@@ -0,0 +1,4 @@
+<p class="snackbar-message">{{ data.message }}</p>
+<button mat-button (click)="snackBarRef.dismissWithAction()">
```

Comment:
we should probably have some flag to show/hide this button if necessary?

Reply:
done